apps/site/api/serializers/audio_serializer.py

```diff
@@ -5,15 +5,15 @@
 from localground.apps.site import models
 
 
-class AudioSerializer(GeometrySerializer):
-    file_name_orig = serializers.CharField(required=False, read_only=True)
```

Comment:
I'm wondering if this field should be deleted. When files are uploaded, it's nice to be able to see the original file name.